lib/tax_cloud/client.rb

```diff
@@ -3,7 +3,7 @@ module TaxCloud #:nodoc:
   class Client < Savon::Client
     # Create a new client.
     def initialize
-      super wsdl: TaxCloud::WSDL_URL
+      super client_params.merge(wsdl: TaxCloud::WSDL_URL)
```

Comment:
The WSDL setting should move into `client_params` as well IMO. There's no need for special treatment. Client params could be:

```ruby
{ wsdl: TaxCloud::WSDL_URL }.merge(TaxCloud.configuration)
```